### PR TITLE
docs: add telemetry page

### PR DESCRIPTION
**Pull request by matskiv**
TSIA :) 

Feedback is welcomed!

### CODE REVIEW

File: docs/pages/telemetry.mdx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+### Why do we collect telemetry
+
+Because DevPod is a freely available open source project, we as maintainers have a very limited idea of how the project is being used, and very limited possibilities to gather this information from the users. Without reliable information, it is difficult to make decisions about the prioritization of features, test automation or bug fixes. Deprecation of the flags and features turns into guesswork, and removal becomes nearly impossible.
+To move forward quickly, and ensure long-term maintainability, we will be making decisions about new features, deprecations, prioritizing test coverage, etc., and we want these decisions to be data-driven.
+
+
+### What are we collecting and how
+
+When a user executes a non-trivial CLI command, one that loads the configuration, the CLI will send an event about the command and basic information about the execution environment (Operating system, CPU architecture, DevPod version, ect.). Another event is sent if a command fails, or when it takes longer than 10 seconds to complete. The events are sent to our self-hosted backend which will store them in the database.
```
Comment (pascalbreuninger) [inline docs/pages/telemetry.mdx:17]: "which will store them in the database" might be an uncesessary detail :D

File: docs/pages/telemetry.mdx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+To move forward quickly, and ensure long-term maintainability, we will be making decisions about new features, deprecations, prioritizing test coverage, etc., and we want these decisions to be data-driven.
+
+
+### What are we collecting and how
+
+When a user executes a non-trivial CLI command, one that loads the configuration, the CLI will send an event about the command and basic information about the execution environment (Operating system, CPU architecture, DevPod version, ect.). Another event is sent if a command fails, or when it takes longer than 10 seconds to complete. The events are sent to our self-hosted backend which will store them in the database.  
+We want to emphasize that we are not interested in collecting data about individuals that are using DevPod, we are collecting only data about how it is being used. This should be evident from the code that implements telemetry on the CLI side, which is of course open source and included in the same source code repository.
```
Comment (pascalbreuninger) [inline docs/pages/telemetry.mdx:18]: let's emphasize this by making it bold 🤭

**Review by pascalbreuninger — COMMENTED** [review]
@matskiv @FabianKramm do we want a telemetry toggle in the UI Settings?

**Comment by FabianKramm** [comment]
@pascalbreuninger I don't think we want something in the UI for now, maybe later

File: docs/pages/telemetry.mdx (hunk 1 above)
Comment (matskiv) [reply docs/pages/telemetry.mdx:17]: True. I mainly trying to say that we are not using some third-party provider of analytics services :) 
I'll remove that bit if I will be making other changes to the PR too.

Comment (pascalbreuninger) [reply docs/pages/telemetry.mdx:17]: ahhh that makes sense. I think you already stated this with "self-hosted backend"

**Pull request merged**